tests/resources/generator/hodgkin_huxley_squid_axon_model_1952/model.dependent.computed.constant.c

```diff
@@ -98,15 +98,17 @@ void computeRates(double voi, double *states, double *rates, double *variables,
 
 void computeVariables(double voi, double *states, double *rates, double *variables, ExternalVariable externalVariable)
 {
+    variables[14] = externalVariable(voi, states, variables, 14);
+    variables[5] = externalVariable(voi, states, variables, 5);
+    variables[6] = variables[5]-10.613;
```

Comment:
How did the previous version of the generator get away with not creating this variable?

Reply:
Yes, I noticed that too. I remember checking it back then and, if I recall correctly, it was the previous analyser (not the previous generator) that had an issue with variables that have a dependency on variables that have been identified as external, and it got fixed while working on this PR.